src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -229,4 +229,45 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('ip_address', help='IPv4 address or CIDR range.')
 
     with self.argument_context('acr check-health') as c:
-        c.argument('ignore_errors', options_list=['--ignore-errors'], help='Ignore errors, displaying them only at the final', action='store_true', required=False)
+        c.argument('ignore_errors', options_list=['--ignore-errors'], help='Ignore errors, not stopping at first error', action='store_true', required=False)
+
+    with self.argument_context('acr scope-map') as c:
+        c.argument('registry_name', options_list=['--registryName', '-r'])
+        c.argument('description', options_list=['--description'], help='Description for the scope map. Max of 140 characters.', required=False)
+        c.argument('add_repository', nargs='+', options_list=['--add'], help='Repositories to be added. Add tuples (repository;action1,action2,...) separated by space.', required=False)
```

Comment:
If there is an enum for different actions thats can be added, I would recommend adding: `arg_type=get_enum_type(<MyEnum>)`